app/game/Main.hs

```diff
@@ -118,6 +108,9 @@ cliParser =
   seed :: Parser (Maybe Int)
   seed = optional $ option auto (long "seed" <> short 's' <> metavar "INT" <> help "Seed to use for world generation")
 
+  ver :: Parser Bool
+  ver = switch (long "version" <> short 'v' <> help "This is the version")
```

Comment:
I think you can safely delete the version command. This is a more discoverable replacement.

If you start with the Version constructor, the compiler should guide you. 🙂